src/output.c

```diff
@@ -382,6 +382,7 @@ new_output_notify(struct wl_listener *listener, void *data)
 	wlr_scene_node_raise_to_top(&output->layer_tree[3]->node);
 	wlr_scene_node_raise_to_top(&output->layer_popup_tree->node);
 	wlr_scene_node_raise_to_top(&output->session_lock_tree->node);
+	wlr_scene_node_raise_to_top(&server->menu_tree->node);
```

Comment:
Should the menu really be on top of the lockscreen?
I assume when you got a menu open and then the lockscreen is started due to swayidle it would allow to execute actions from that menu.

Reply:
Good point. Didn't think of that. Fixed.